15_Ansible_test/a_var.yml

```diff
@@ -0,0 +1,10 @@
+$ANSIBLE_VAULT;1.1;AES256
+36653132333237323066323231363839336365373435376232326234313331303764383863346430
+3364333666316333373632623237633261323633313663300a363137306537626566663133373564
+66353065313063633862306465323662303661663235383331386639303839613630623132646534
+3230336239366436300a613430333130353137323862313830643132646265376663373934653061
+66356363346135343265346235653830623563383833393765666564323434393261663838643331
+64313864383363636231356663633561343033353161666339646530636266393031373533363730
+65303466323339353337333762376362666232333937626539623062626533323263636631313563
+61636138343966613436623932323237633463373766383138306639353564363466303666636131
+3734
```

Comment:
Could you please explain what is the value of these variables? Do not print the plain text if this is credentials, just comment, please.